content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -44,6 +44,15 @@ Example outbound CodeableConcept displaying both standard and proprietary codes:
       }
     }
 
+## Documents
+
+<!-- use html header to avoid showing up in toc -->
+<h3>DocumentReference</h3>
+
+The DocumentReference Resource supports proprietary codes for:
+
+* DocumentReference.type codes are maintained in [Code Set 72 Event Code](#code-set-72-event-code)
```

Comment:
can this be Code Set 72 Event Codes?

Reply:
[Updated](https://github.com/cerner/fhir.cerner.com/pull/334/commits/5e38b77a2b8751100b9950d56d54324d1e102e2b#diff-6fdcb00b5b1b6edd51b687794bea6b95R54)

Also, the Documents header was one level too high, [fixed it here](https://github.com/cerner/fhir.cerner.com/pull/334/commits/5e38b77a2b8751100b9950d56d54324d1e102e2b#diff-6fdcb00b5b1b6edd51b687794bea6b95R47)

Reply:
can we change the anchor as well?

Reply:
Actually, it looks like I copied the dstu2 link instead of r4.  I've [updated](https://github.com/cerner/fhir.cerner.com/pull/334/commits/aa34333db1d8bd104a3fa21a45fcc97abcf2dbd0#diff-6fdcb00b5b1b6edd51b687794bea6b95R54) it to correctly link to the existing event code on the r4 page.